wftests/warrior_tests/testcases/common_actions_tests/test_slurp_json.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
+<Testcase>
+        <Details>
+                <Name>test_slurp_json.xml</Name>
+                <Title>test_slurp_json</Title>
+                <Engineer>Sekhar</Engineer>
+                <Date>2018-11-30</Date>
+                <Time>12:08</Time>
+                <State/>
+                <InputDataFile>No_Data</InputDataFile>
+                <Datatype/>
+                <default_onError action="next"/>
+                <Logsdir/>
+                <Resultsdir/>
+                <ExpectedResults/>
+        </Details>
+        <Requirements>
+                <Requirement/>
+        </Requirements>
+        <Steps>
+                 <step Driver="common_driver" Keyword="store_in_repo" TS="1" draft="no">
+                        <Arguments>
+                                <argument name="filepath" value="slurp_content.json"/>
```

Comment:
Is the filepath value being stored in ``` slurp_content.json ```? If so, then where is ``` slurp_content.json ``` being initialized?

Reply:
I am not sure if I fully understand what ``` slurp_content.json ``` is.

Reply:
I will suggest moving slurp_content.json into the config_files directory.

Reply:
done